Packages/com.chisel.components/Chisel/Components/API.public/BrushMeshAssetFactory/LinearStairs/BrushFactory.LinearStairs.cs

```diff
@@ -214,8 +214,13 @@ public static bool GenerateLinearStairsSubMeshes(CSGBrushSubMesh[] subMeshes, CS
 					case StairsSideType.None: rightSideType = StairsSideType.Down; break;
 				}
 			}
-			var boundsMin			= definition.bounds.min;
-			var boundsMax			= definition.bounds.max;
+			var boundsMin = definition.bounds.min;
+			var boundsMax = definition.bounds.max;
+
+            if (boundsMin.y > boundsMax.y) { var t = boundsMin.y; boundsMin.y = boundsMax.y; boundsMax.y = t; }
+            if (boundsMin.x > boundsMax.x) { var t = boundsMin.x; boundsMin.x = boundsMax.x; boundsMax.x = t; }
+            if (boundsMin.z > boundsMax.z) { var t = boundsMin.z; boundsMin.z = boundsMax.z; boundsMax.z = t; }
```

Comment:
This makes negative sized stairs still work properly (before it would be clamped to 0)
In a future PR I want the stairs to flip direction when scaled in z direction